Feature/m2 init project

andrew-gurzhi opened the pull request:
1 Задание Инициализация проекта, создан подпроект common-backend, написаны тесты

Commit:
add commond backend module, add test

Commit:
add kotest library and test

Commit:
* Исправлены файлы билдов
- Финальные правки

* Исправлены файлы билдов
- Финальные правки

svok commented on .idea/.gitignore:
```diff
@@ -0,0 +1,8 @@
+/../../../../../../:\Users\gurzh\IdeaProjects\otus-kotlin-rentalproperty\.idea/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Я обычно игнорю всю директорию .idea. Она нормально настраивается userwide, специфичные для проекта настройки предпочитаю не делать - слишком много проектов и настраивать каждый - это мучение

svok commented on build.gradle.kts:
```diff
@@ -0,0 +1,15 @@
+plugins {
+    val kotlinVersion = "1.4.10"
```
Версию рекомендую вынести в gradle.properties

svok commented on settings.gradle.kts:
```diff
@@ -0,0 +1,2 @@
+rootProject.name = "otus-kotlin-rentalproperty"
+include("common-backend")
```
По названию подпроектов см. https://github.com/orgs/otuskotlin/teams/students-202012/discussions/2

svok reviewed (commented):
Минимум вы конечно же сделали, так что никаких вопросов.
Но рекомендую еще тренироваться на каких-то примерах, задачах.

Commit:
Исправление замечаний
* Переименовано название подпроект
- версия kotlinVersion добавлена в gradle.properties
- ./idea добавлена в гитигнор
